src/licensedcode/data/rules/bsd-new-oracle-glassfish.RULE

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) YYYY Oracle and/or its affiliates. All rights reserved.
```

Comment:
It is best to avoid including copyright in license rules. Can you remove that line and do the same in other rules? Copyright when not part of the license text proper introduce a bias and possible false positive detection. They will be caught by the copyright detection engine alright in any case.